src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+interface Props {
+  todos: Todo[],
+  removeTodo: (id: number) => void,
+  changeStatus: (id: number, property: Partial<Todo>) => void,
+}
+export const TodoList: FC<Props> = ({ todos, removeTodo, changeStatus }) => {
+  return (
+    <section className="todoapp__main">
```

Comment:
Wrap the list of todos in a `<ul>` tag with a className of `todo-list` for better accessibility and alignment with the expected structure 📝